src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -22,7 +22,12 @@ class FormStartControls extends React.Component {
     this.props.router.push(this.props.startPage);
   };
 
+  captureAnalytics = () =>
+    this.props.gaStartEventName &&
```

Comment:
Could we use the `formId` for this instead (or like a `start-form-${formId}`)? Is there a reason we'd want the name to be different than the formID?

Reply:
I can't think of a reason that wouldn't work. I think I did this because I wasn't sure if I had access to the form ID, but if I recall, we do...

I'll look into it. :+1: